Sources/Hummingbird/Router/TrieRouter.swift

```diff
@@ -93,20 +93,62 @@ struct RouterPathTrieBuilder<Value: Sendable> {
     }
 }
 
-/// Trie used by Router responder
-struct RouterPathTrie<Value: Sendable>: Sendable {
-    let root: Node
+import NIOCore
+
+public struct BinaryRouterResponder<Context: BaseRequestContext>: HTTPResponder {
+    let trie: BinaryTrie<EndpointResponders<Context>>
```

Comment:
This seems like an odd place to add this type. Might be worthwhile adding its own file. The original `RouterResponder` has its own file.